--- httpstan/services_stub.py  2024-01-01 00:00:00.000000000 +0000
+++ httpstan/services_stub.py  2024-01-01 00:00:00.000000000 +0000
@@ -1,6 +1,9 @@
 import concurrent.futures
 import multiprocessing as mp
+import sys
+
+ctx = mp.get_context("spawn" if sys.platform == "win32" else "fork")

-executor = concurrent.futures.ProcessPoolExecutor(mp_context=mp.get_context("fork"))
+executor = concurrent.futures.ProcessPoolExecutor(mp_context=ctx)